docs/source/admin/fs/rest.rst

```diff
@@ -193,6 +193,18 @@ There is a specific id named ``_auto_`` where the ID will be
 autogenerated by elasticsearch. It means that sending twice the same
 file will result in 2 different documents indexed.
 
+Additional tags
+~~~~~~~~~~~~~~~
+
+Add custom tags to the document. In case you want to do filtering on those tags (examples are projectId or tenantId). These tags are assigned to meta.raw
```

Comment:
I've somthing in place to merge the tags into the document.
It's possible to overwrite the document (like overwriting content/meta etc.), but what shall I do with the response object. The result of the merge can't be serialized back to the Doc object, since the object can be more extended.
Any suggestions?

Reply:
I see. :( 

So may be we should introduce a new field name instead of `meta.raw` like `external` which would be defined as a `Map<String, Object>` in `Doc` class?

Can't come with a better idea for now.